### DISCUSSION TITLE
[i3-gaps] i get this annoying splashscreen kind of on any display change like new tabs , change workspace , loading page

### DISCUSSION THREAD

**Discussion by OddSteak** [i3/i3#4946] [Q&A]
im a noob please tell what more info is needed

video : https://user-images.githubusercontent.com/84833531/163043240-d86e9906-d0f6-4b0f-a9d8-81be3e2b7498.mp4
i3 config : http://0x0.st/ocPQ.txt
picom.conf : http://0x0.st/ocP1.conf

i3-gaps version : 4.19.1

picom version : 9.1
OS: fedora 35 i3 spin
X.Org X Server 1.20.14
X Protocol Version 11, Revision 0

**Comment by orestisfl** [comment]
Does it happen without picom?

**Reply by OddSteak** [reply]
it's worse without picom  now i hv installed arch linux and am watching it on my every step it only happens on selecting blank page for new tab on firefox firefox home doesn't cause the problem
also these lines in my config were causing other display related issues 
`for_window [window_role="GtkFileChooserDialog"] resize set 800 600`
`for_window [window_role="GtkFileChooserDialog"] move position center`
is there something wrong with them ?they do succeed in repositioning and resizing file chooser dialog but cause other issues in firefox like i get random boxes of wallpapers and that splashscreen kind of when i open another app beside firefox
here's my new i3 config: http://0x0.st/oclI.txt
picom config is the same as before
LOG:
https://logs.i3wm.org/logs/6210919730249728.bz2
firefox is in workspace 1 here
